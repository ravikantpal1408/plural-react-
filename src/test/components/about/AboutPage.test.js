import React from 'react';
import AboutPage from '../../../components/about/AboutPage';
import renderer from 'react-test-renderer';

// import { mount, shallow } from 'enzyme';

describe('About Page Component', () => {
  it('should match with snapshot ', () => {
    const wrapper = renderer.create(<AboutPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
