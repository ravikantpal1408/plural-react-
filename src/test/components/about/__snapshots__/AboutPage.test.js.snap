// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page Component should match with snapshot  1`] = `
<div>
  <h2>
    About
  </h2>
  <p>
    This app uses React, Redux, React Router, and many other helpful libraries.
  </p>
</div>
`;
